.github/ISSUE_TEMPLATE.md

```diff
@@ -1,8 +1,3 @@
-### Note
-
-
-Does your post belong on Github?  If you are reporting a bug or issue, or are contributing to the development of Kolibri, then it belongs here!  If you are seeking advice on technical and non-technical elements of deploying KA Lite and Kolibri in your community, you'll find relevant information at our [community forum](https://community.learningequality.org). 
-
```

Comment:
Moved to CONTRIBUTING.md

My reason is: It's where it belongs, otherwise EVERY time you post an issue, you have to delete text, and the action of removing that might prompt people to do CTRL+A -> DEL, which is already a big issue for our issues :)